src/main/java/picard/illumina/CollectIlluminaBasecallingMetrics.java

```diff
@@ -38,26 +38,39 @@
 import picard.illumina.parser.ClusterData;
 import picard.illumina.parser.IlluminaDataProviderFactory;
 import picard.illumina.parser.IlluminaDataType;
+import picard.illumina.parser.IlluminaFileUtil;
+import picard.illumina.parser.ParameterizedFileUtil;
 import picard.illumina.parser.ReadStructure;
+import picard.illumina.parser.readers.AbstractIlluminaPositionFileReader;
 import picard.illumina.parser.readers.BclQualityEvaluationStrategy;
+import picard.illumina.parser.readers.LocsFileReader;
 import picard.util.TabbedTextFileWithHeaderParser;
 
 import java.io.File;
 import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 import java.util.SortedMap;
 import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static picard.illumina.NewIlluminaBasecallsConverter.getTiledFiles;
```

Comment:
I think we should avoid static imports like this and instead call the method using `NewIlluminaBasecallsConverter.getTiledFiles()`